go.mod

```diff
@@ -22,4 +22,5 @@ require (
 	golang.org/x/text v0.23.0 // indirect
 	google.golang.org/genproto/googleapis/api v0.0.0-20240826202546-f6391c0de4c7 // indirect
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20240903143218-8af14fe29dc1 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
```

Comment:
This remaining indirect one comes from github.com/stretchr/testify. PR at https://github.com/stretchr/testify/pull/1772